Essentials/src/com/earth2me/essentials/Teleport.java

```diff
@@ -72,6 +72,18 @@ public void cooldown(boolean check) throws Exception {
         }
     }
 
+    private void border(Location loc, IUser teleportee) throws Exception {
+        // TODO: review: should we check if the world is null? I mean I kind of assume a location always has a world.
+        if (!loc.getWorld().getWorldBorder().isInside(loc)
```

Comment:
Locations are always assumed to have a valid world afaik. If they didn't we would have some other problems.